src/test/build/shared-build-targets-utils/Utils/DotNetCli.cs

```diff
@@ -32,7 +32,8 @@ public Command Exec(string command, params string[] args)
                 newArgs.Insert(0, "-v");
             }
 
-            return Command.Create(Path.Combine(BinPath, $"dotnet{Constants.ExeSuffix}"), newArgs);
+            return Command.Create(Path.Combine(BinPath, $"dotnet{Constants.ExeSuffix}"), newArgs)
+                .EnvironmentVariable("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "1");
```

Comment:
It seems like we should avoid this in any cases where we are using an xcopy CLI.